app/Providers/AppServiceProvider.php

```diff
@@ -11,9 +14,41 @@ class AppServiceProvider extends ServiceProvider
      *
      * @return void
      */
-    public function boot()
+    public function boot(): void
     {
-        //
+        /**
```

Comment:
Please move all this methods to the HelpersServiceProvder class of the LravelHelpers package. Instead VersionEnum it sould use VersionEnumContract (new interface in the package)